airbyte-cdk/python/airbyte_cdk/sources/streams/core.py

```diff
@@ -295,7 +348,7 @@ def stream_slices(
         :param stream_state:
         :return:
         """
-        return [None]
+        return [{}]  # I really hate that this used to be None, which was just cruft to get the first loop in
```

Comment:
I agree. Do you want to keep the comment in code for posterity or can we remove it?

Reply:
removing